integration_tests/e2e/overview/educationAndTrainingTab.cy.ts

```diff
@@ -27,6 +27,24 @@ context('Prisoner Overview page - Education And Training tab', () => {
       .hasFunctionalSkillsDisplayed()
   })
 
+  it('should display Education and Training data given curious API returns a 404', () => {
+    // Given
+    cy.task('stubLearnerProfile404Error')
+
+    cy.signIn()
+    const prisonNumber = 'G6115VJ'
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    const overviewPage = Page.verifyOnPage(OverviewPage)
+
+    // When
+    overviewPage.selectTab('Education and training')
+
+    // Then
+    overviewPage //
+      .activeTabIs('Education and training')
+      .hasFunctionalSkillsDisplayed()
```

Comment:
Of course this doesn't assert there's no data, but I think it's fine. I was wondering if the cheerio tests (still a sh!t name!) could assert this instead, but the main thing is that the expected sections are displayed 👍